handlers/filter_page.go

```diff
@@ -0,0 +1,68 @@
+package handlers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/ONSdigital/dp-frontend-dataset-controller/config"
+	"github.com/ONSdigital/log.go/v2/log"
+	"github.com/gorilla/mux"
+
+	dpDatasetApiSdk "github.com/ONSdigital/dp-dataset-api/sdk"
+)
+
+// FilterPageHandler handles requests to /filters/{filterID}
+func FilterPageHandler(f FilterClient, datasetClient DatasetAPISdkClient) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		vars := mux.Vars(r)
+		cfg, err := config.Get()
+
+		headers := dpDatasetApiSdk.Headers{
+			CollectionID:         "",
+			DownloadServiceToken: "",
+			ServiceToken:         "",
+			UserAccessToken:      "",
+		}
+
```

Comment:
Empty line? Errors are handled immediately in Go, no blank lines between logic and error handling